src/table/TableHeadWrapper.jsx

```diff
@@ -32,7 +32,10 @@ export default function TableHeadWrapper({
   selectionsAPI,
   focusedCellCoord,
 }) {
-  const classes = useStyles(getHeadStyle(layout, theme));
+  const classes = useStyles(
+    useMemo(() => getHeadStyle(layout, theme)),
+    [layout, theme.name()]
+  );
```

Comment:
I think you should save the head style outside of `useStyles`, the same way we do for body

Reply:
okay, look into it

Reply:
do you mean 
`const styling = useMemo(() => getBodyStyle(layout, theme), [layout, theme.name()]);
  const classes = useStyles(styling);`
in body?

Reply:
Yes, just to avoid any weird stuff when nesting hooks like that